### PR TITLE
Release v0.2.0

### PR DESCRIPTION
**Pull request by jakexks**
- Add serviceAccount to the all-in-one deployment + regenerate without redundant prefix
- Use more idiomatic event reasons
- Send events from the Issuer controller

### CODE REVIEW

File: config/rbac/kustomization.yaml [hunk 1]
```diff
@@ -3,10 +3,11 @@ resources:
...
 # the auth proxy (https://github.com/brancz/kube-rbac-proxy)
 # which protects your /metrics endpoint.
-- auth_proxy_service.yaml
-- auth_proxy_role.yaml
-- auth_proxy_role_binding.yaml
-- auth_proxy_client_clusterrole.yaml
+#- auth_proxy_service.yaml
+#- auth_proxy_role.yaml
+#- auth_proxy_role_binding.yaml
+#- auth_proxy_client_clusterrole.yaml
```
Comment (wallrj) [inline config/rbac/kustomization.yaml:13]: Does this mean the /metrics endpoint is now unprotected?

File: deploy/google-cas-issuer.yaml [hunk 2]
```diff
@@ -462,7 +412,7 @@ spec:
         - --zap-devel=true
         command:
         - /google-cas-issuer
-        image: quay.io/jetstack/google-cas-issuer:latest
+        image: quay.io/jetstack/cert-manager-google-cas-issuer:latest
```
Comment (wallrj) [inline deploy/google-cas-issuer.yaml:415]: I don't see any related changes to the build scripts in this PR. Was the Docker image name changed somewhere else?

**Review by wallrj — APPROVED** [review]
A couple of questions, but  please unhold when those are answered.

/lgtm
/hold

File: deploy/google-cas-issuer.yaml (hunk 2 above)
Comment (jakexks) [reply deploy/google-cas-issuer.yaml:415]: This was a mistake accidentally included from another branch where I am writing an e2e test suite

File: config/rbac/kustomization.yaml (hunk 1 above)
Comment (jakexks) [reply config/rbac/kustomization.yaml:13]: Yes and no, as it also removes the service that pointed to the metrics endpoint so it's not reachable at all. Consuming this in a prometheus rule would probably be good to gate behind a value in an uncoming helm chart (there's another PR for that), or if someone was to look at the kustomization themselves they would be able to uncomment it and re-add it.

**Pull request merged**